pages/shared/groupWorkInitial.ejs

```diff
@@ -0,0 +1,33 @@
+<div class="container-fluid">
+  <div class="row">
+      <% if (permissions.student_authz_create) { %>
+      <div class="col-sm bg-light py-4 px-4 border">
+        <form id="create-form" name="create-form" method="POST">
+          <input type="text" class="form-control" name="groupName" placeholder="Group Name">
+          <small id="groupNameHelp" class="form-text text-muted">*Alpha and numeric characters only</small>
```

Comment:
Follow the layout here https://getbootstrap.com/docs/4.5/components/forms/#help-text for help text. Note the `aria-describedby` to link the input element to the help text for screen readers.